Photon-Tinker/LogList.swift

```diff
@@ -0,0 +1,132 @@
+//
+// Created by Raimundas Sakalauskas on 11/06/18.
+// Copyright © 2018 Particle. All rights reserved.
+//
+
+import UIKit
+import Crashlytics
```

Comment:
What Crashlytics components are being used in this?  I'm not seeing anything that sticks out.

Reply:
```